device/arista/x86_64-arista_7060dx5_32/Arista-7060DX5-32-25Gx96-100Gx8-200Gx8/port_config.ini

```diff
@@ -0,0 +1,114 @@
+#name          lanes              alias        index  speed     fec
+Ethernet0      1                  Ethernet1/1  1      25000
```

Comment:
we should make 25G fec with fc,  100G  and 200G with rs fec.

Reply:
For 25G, FC looks not supported by the switch chip from the return of SAI_PORT_ATTR_SUPPORTED_FEC_MODE (NONE and RS supported). We will further check if it is hardware constraint or a software issue that we can fix. We did not specify it here to use the default value from switch chip (which is NONE for 25G) . User can configure/override the FEC config by /etc/sonic/config_db.json, as long as it is supported.

Reply:
thanks, can you please check with the vendor?  I am concerned even if we do it later by configDB, it will hit the same issue.

Reply:
It is confirmed that FC FEC is not supported by TH4 chip.